src/UiTPAS/Event/Event/EventCardSystemsUpdatedDeserializer.php

```diff
@@ -34,7 +33,7 @@ public function deserialize(StringLiteral $data)
             throw new \InvalidArgumentException('Expected cardSystems property to be an array.');
         }
 
-        $cardSystems = new CardSystems();
+        $cardSystems = [];
         foreach ($dto->cardSystems as $cardSystemDTO) {
```

Comment:
This `foreach` can now be changed to `array_map` if I'm not mistaken?

Reply:
`array_map` to produce an array indexed by specific keys looks pretty messy in my opinion. I'd rather leave it as a `foreach` in that case.

Reply:
Ah I did not notice the keys by id. In that case `foreach` is fine yeah.